spec/features/access_points/dissertation_theses_spec.rb

```diff
@@ -30,4 +30,23 @@
       expect(page).to have_css('h3', text: 'Date')
     end
   end
+
+  it 'Toggles the Stanford work facet' do
+    expect(page).to have_css('#facet_option_stanford_only', visible: :visible)
+
+    checkbox = find_by_id('facet_option_stanford_only')
+
+    checkbox.click
+
+    checked = find_by_id('facet_option_stanford_only')
+    expect(page).to have_css('.filter-stanford_work_facet_hsim')
+    expect(checked.checked?).to be(true)
```

Comment:
I'd prefer a check that shows this actually filters documents in/out from the result set.  Checking that "checked" is set can be tested in a component spec with much less overhead than a integration test.